WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -247,6 +247,12 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
 
         webView.isP2 = post.isP2Type()
 
+        if post.content?.hasSuffix("[…]") == true {
```

Comment:
There gotta be a better way to test, but I couldn't find one. I think it'll do for now.

Reply:
Hey, @nbradbury, would you mind checking how the scenario from the PR is detected in Android?

Reply:
On Android, when the post is tapped in the list view we show the excerpt in the detail view, with a link at the bottom to visit the site for the full post.

![reader](https://github.com/user-attachments/assets/4b725fe1-7ac2-427a-b329-6cf8bcd7dd45)